test/MUIDataTableToolbar.test.js

```diff
@@ -124,6 +124,22 @@ describe('<TableToolbar />', function() {
     assert.strictEqual(actualResult.length, 0);
   });
 
+  it('should render a toolbar with custom TableSearch when option.customTableSearch is method ', () => {
```

Comment:
Let's change the wording here to 'should render a toolbar with custom TableSearch when option.customTableSearch is provided' (`provided` instead of `method`). The test itself documents that what we're looking for is a React component. `method` is a little confusing to me.